File: csv/octave/delete/csv_delete.m
#! /usr/bin/octave -qf
#
#	csv_delete.m
#
#					Nov/11/2014
#
# ------------------------------------------------------------------
path (path, "/var/www/data_base/common/octave_common");
# ------------------------------------------------------------------
printf ("*** 開始 ***\n")
arg_list = argv ();
file_csv = arg_list{1};
key_in = arg_list{2};
disp (file_csv)
disp (key_in)
#
dict_aa = struct;
dict_aa = csv_read_proc (file_csv)
#
if  (isfield (dict_aa, key_in))
	dict_aa = rmfield (dict_aa, key_in)
	dict_display_proc (dict_aa)
	csv_write_proc (file_csv,dict_aa)
endif
#
printf ("*** 終了 ***\n")
# ------------------------------------------------------------------

File: csv/jython/delete/csv_delete.py
#! /usr/bin/jython
# -*- coding: utf-8 -*-
#
#	delete/csv_delete.py
#
#					May/27/2015
import	sys
import	string
#
# ---------------------------------------------------------------
sys.path.append ('/var/www/data_base/common/jython_common')
from jython_text_manipulate import csv_read_proc
from jython_text_manipulate import csv_write_proc
from jython_text_manipulate import dict_delete_proc
# ---------------------------------------------------------------
#
print ("*** 開始 ***")
#
file_in = sys.argv[1]
id_in = sys.argv[2]
print ("%s" % id_in)

dict_aa = csv_read_proc	(file_in)

dict_bb = dict_delete_proc (dict_aa,id_in)

csv_write_proc (file_in,dict_bb)
#
print ("*** 終了 ***")
# ---------------------------------------------------------------
